operations/services.md

```diff
@@ -214,6 +216,134 @@ Updates the number of available resources in [Resource category](enterprises.md#
 {}
 ```
 
+## Add availability blocks
+
+Adds availability blocks which are used to group related [Availability update](#availability-update)s. This makes limiting public availability easier and more organized. After creation it can be referenced from [Update availability](#update-service-availability) operation using `AvailabilityBlockId` parameter. They can also be used to group [Reservation](reservations.md#reservation)s.
+
```

Comment:
minor edits for ease of reading:

Adds availability blocks which are used for group-related [Availability update](#availability-update)s. This makes limiting public availability easier and more organized. After creation, it can be referenced from [Update availability](#update-service-availability) operation using `AvailabilityBlockId` parameter. They can also be used for group [Reservation](reservations.md#reservation)s.